Defra.Cdp.Backend.Api/Endpoints/ActionsEndpointV2.cs

```diff
@@ -0,0 +1,20 @@
+using Defra.Cdp.Backend.Api.Models;
+using Defra.Cdp.Backend.Api.Services.Actions;
+
+namespace Defra.Cdp.Backend.Api.Endpoints;
+
+public static class ActionsEndpointV2
+{
+    public static void MapActionsEndpoint(this IEndpointRouteBuilder app)
+    {
+        app.MapGet("/actions/latest-config/{environment}", LatestAppConfig);
```

Comment:
Wondering if it would be better if this was ConfigEndpoints or something?
`/config/latest/env `
Not sure we need to indicate its from a gh action run once we have the data.